sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSourceType.cs

```diff
@@ -42,16 +42,6 @@ namespace Azure.AI.MetricsAdvisor.Models
         /// </summary>
         public static DataFeedSourceType AzureTable { get; } = new DataFeedSourceType(AzureTableValue);
 
-        /// <summary>
-        /// Elasticsearch.
-        /// </summary>
-        internal static DataFeedSourceType Elasticsearch { get; } = new DataFeedSourceType(ElasticsearchValue);
-
-        /// <summary>
-        /// HTTP Request.
-        /// </summary>
-        internal static DataFeedSourceType HttpRequest { get; } = new DataFeedSourceType(HttpRequestValue);
-
```

Comment:
Not supported by the service anymore. We had already made them internal in a previous PR, but we can safely remove it now since it's not used by CodeGen.